.github/workflows/run-kubecover-api-manager.yaml

```diff
@@ -0,0 +1,23 @@
+name: API Manager Kubecover Flow
+
+on: [pull_request]
```

Comment:
I don't want to run automated tests of Kubecover during build of api manager, until we are 100% sure the tests are not flapping.